components/tikv_util/src/store/region.rs

```diff
@@ -38,6 +38,18 @@ pub fn region_on_same_stores(lhs: &Region, rhs: &Region) -> bool {
     })
 }
 
+/// Check if the given region exists on stores.
+pub fn region_on_stores(region: &Region, store_ids: &Vec<u64>) -> bool {
```

Comment:
This is a bit weird for me, could we just check the `self store id` in `store_ids`?

Reply:
Then we dont need this fn

Reply:
Precisely, this function is used to check and filter out regions neither exists on `self store_id` nor `abnormal store_ids`. That is, we just wanna wake up hibernated regions both existing in `abnormal stores` and held by `self store_id`.

Reply:
And I've update and polish the related annotations to reduce your confusions.